components/about-page/about-founder/AboutFounder.js

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import { getString, renderRichText } from '../../../lib/richText';
```

Comment:
Nit: seems like not related to this PR also, but the navigation on hover looks strange for the page you are on (because it applies the bold + the hover)
<img width="205" alt="Screen Shot 2021-05-09 at 5 07 59 PM" src="https://user-images.githubusercontent.com/33647880/117586930-48ca0600-b0e9-11eb-87b4-72a2fbdf09c7.png">

Reply:
fixed